docs/install_guides/installing-openvino-docker-linux.md

```diff
@@ -122,245 +104,135 @@ RUN yum update -y && yum install -y https://dl.fedoraproject.org/pub/epel/epel-r
-```sh
-WORKDIR /tmp
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-        libboost-filesystem1.65-dev \
-        libboost-thread1.65-dev \
-        libjson-c3 libxxf86vm-dev && \
-    rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
-```
-   - **Ubuntu 20.04**:
-```sh
-WORKDIR /tmp
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-        libboost-filesystem-dev \
-        libboost-thread-dev \
-        libjson-c4 \
-        libxxf86vm-dev && \
-    rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
-```
-   - **CentOS 7**:
-```sh
-WORKDIR /tmp
-RUN yum update -y && yum install -y \
-        boost-filesystem \
-        boost-thread \
-        boost-program-options \
-        boost-system \
-        boost-chrono \
-        boost-date-time \
-        boost-regex \
-        boost-atomic \
-        json-c \
-        libXxf86vm-devel && \
-    yum clean all && rm -rf /var/cache/yum
-```
-3. Run `hddldaemon` on the host in a separate terminal session using the following command:
-```sh
-$HDDL_INSTALL_DIR/hddldaemon
-```
+- Intel® Neural Compute Stick 2 device changes its VendorID and DeviceID during execution and each time looks for a host system as a brand new device. It means it cannot be mounted as usual.
+- UDEV events are not forwarded to the container by default, and it does not know about the device reconnection. The prebuilt Docker images and provided Dockerfiles include `libusb` rebuilt without UDEV support.
+- Only one NCS2 device connected to the host can be used when running inference in a container.
 
-### Run the Docker* Image for Intel® Vision Accelerator Design with Intel® Movidius™ VPUs
 
-To run the built Docker* image for Intel® Vision Accelerator Design with Intel® Movidius™ VPUs, use the following command:
+### Running the Image on Intel® Vision Accelerator Design with Intel® Movidius™ VPUs
 
+Use the following command:
 ```sh
 docker run -it --rm --device=/dev/ion:/dev/ion -v /var/tmp:/var/tmp <image_name>
 ```
 
-> **NOTES**:
->
-> - The device `/dev/ion` needs to be shared to be able to use ion buffers among the plugin, `hddldaemon` and the kernel.
-> - Since separate inference tasks share the same HDDL service communication interface (the service creates mutexes and a socket file in `/var/tmp`), `/var/tmp` needs to be mounted and shared among them.
+Note the following things: 
+* The device `/dev/ion` needs to be shared to be able to use ion buffers among the plugin, `hddldaemon` and the kernel.
+* Since separate inference tasks share the same HDDL service communication interface (the service creates mutexes and a socket file in `/var/tmp`), `/var/tmp` needs to be mounted and shared among them.
 
-In some cases, the ion driver is not enabled (for example, due to a newer kernel version or iommu (Input-Output Memory Management Unit) incompatibility). `lsmod | grep myd_ion` returns empty output. To resolve, use the following command:
 
+#### If the ion Driver is Not Enabled
```

Comment:
Can you please add the note below here? This is a problem that users can sometimes face.

If the application runs inference of a network with a big size(>4MB) of input/output, the HDDL plugin will use shared memory.  
You should mount `/dev/shm` as volume in this case:
```bash
docker run -it --rm --device=/dev/ion:/dev/ion -v /var/tmp:/var/tmp -v /dev/shm:/dev/shm <image_name>
```
Also, in some cases you can encounter a permission error for files in `/dev/shm` (see `hddldaemon.log`). The possible cause is uid and gid of the container user are different from uid and gid of the user which had created `hddldaemon` service on the host.  
You can try one of these solutions:
* create the user in the docker container with uid and gid which are the same with the HDDL daemon's user 
* set container user umask to 000: `umask 000`
* start HDDL daemon on the host as root and start container as root `docker run -u root:root` (NOT RECOMMENDED)

Reply:
Can it go to the Troubleshooting guide? I understand you want related content to be together, but I think in the installation steps we should keep it clear and short. If there are common issues users might encounter very often, we can add a note to warn them and try to improve from the product itself. If the issue is not that common, I suggest that we put it in the Troubleshooting Guide. Does it make sense? What severity of this issue is?

Reply:
This is rather not a issue, but a known specificity of interaction between HDDL device and Docker container. Can we divide this into two parts (put the text below in the "Running" and place the rest of the text in the Troubleshooting guide)?

----
If the application runs inference of a network with a big size(>4MB) of input/output, the HDDL plugin will use shared memory. You should mount `/dev/shm` as volume in this case:
```bash
docker run -it --rm --device=/dev/ion:/dev/ion -v /var/tmp:/var/tmp -v /dev/shm:/dev/shm <image_name>
```

Reply:
Thanks. Updated. Please have a check. :)